<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextInputOverlay</class>
 <widget class="QWidget" name="TextInputOverlay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>153</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="questionLabel">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string notr="true">Question</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="auxLabel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLineEdit" name="responseBox">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="errorLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="cancelButton">
               <property name="text">
                <string>Cancel</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/libentertaining/icons/dialog-cancel.svg</normaloff>:/libentertaining/icons/dialog-cancel.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="okButton">
               <property name="text">
                <string>OK</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/libentertaining/icons/dialog-ok.svg</normaloff>:/libentertaining/icons/dialog-ok.svg</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Keyboard" name="keyboardWidget" native="true"/>
   </item>
   <item>
    <widget class="GamepadHud" name="gamepadHud" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GamepadHud</class>
   <extends>QWidget</extends>
   <header location="global">gamepadhud.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Keyboard</class>
   <extends>QWidget</extends>
   <header>keyboards/keyboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>responseBox</sender>
   <signal>returnPressed()</signal>
   <receiver>okButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
